routing/blinding.go

```diff
@@ -70,3 +72,80 @@ func (b *BlindedPayment) Validate() error {
 
 	return nil
 }
+
+// toRouteHints produces a set of chained route hints that represent a blinded
+// path. This function assumes that we have a valid blinded path (one with at
+// least two blinded hops / encrypted data blob).
+func (b *BlindedPayment) toRouteHints() map[route.Vertex][]*channeldb.CachedEdgePolicy { //nolint:lll
+	hintCount := len(b.BlindedPath.BlindedHops) - 1
+	hints := make(
+		map[route.Vertex][]*channeldb.CachedEdgePolicy, hintCount,
+	)
+
+	// Start at the unblinded introduction node, because our pathfinding
+	// will be able to locate this point in the graph.
+	fromNode := route.NewVertex(b.BlindedPath.IntroductionPoint)
+
+	features := lnwire.EmptyFeatureVector()
+	if b.Features != nil {
+		features = b.Features.Clone()
+	}
+
+	// Use the total aggregate relay parameters for the entire blinded
+	// route as the policy for the hint from our introduction node. This
+	// will ensure that pathfinding provides sufficient fees/delay for the
+	// blinded portion to the introduction node.
```

Comment:
> we treat the entire blinded route as a single "hop hint" and the existing pathfinding can handle this correctly without much modification. Is that the idea?

Basically, but it's a chain of hop hints rather than a single hint. 
Eg for some blinded path: Intro -> B1 -> B2
We have hints: (Intro -> B1 / B1 -> B2)